### PR TITLE
add support for Member.pending

### LINKED ISSUE #30: autorole and pending members
[Will be included in d.py 1.6.0.](https://discordpy.readthedocs.io/en/latest/api.html#discord.Member.pending)

When a user confirms themselves with membership screening, `on_member_update()` gets called and `Member.pending` becomes `False`.

will require some modifications to current code:
```
member join event:
    if not member.pending:
        do autoroles
member update event:
    if member.pending -> not member.pending:
        do autoroles
```

### PR DESCRIPTION
**Pull request by classabbyamp**
fixes #30

### CODE REVIEW

**Comment by classabbyamp** [comment]
~~This is blocked until the release of discord.py 1.6.0~~

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ and this project adheres (somewhat) to [Semantic Versioning](https://semver.org/
 ### Added
 - Ability to exclude bots from autorole (on by default).
 - `mb?rolestats` command.
+- Support for membership screening to autorole.
+### Changed
+- Bumped version of discord.py to 1.6.0
```
Comment (0x5c) [inline CHANGELOG.md]: Should this say "discord.py dependency" instead?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ A small bot for basic guild stuff.
 
 Includes:
 - Autorole
+    - Supports [Membership Screening](https://support.discord.com/hc/en-us/articles/1500000466882-Rules-Screening-FAQ)
```
Comment (0x5c) [inline README.md:9]: add "out of the box"

**Review by 0x5c — APPROVED** [review]

File: dev-requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 -r requirements.txt
-discord.py-stubs==1.5.0
+discord.py-stubs~=1.5.0
```
Comment (classabbyamp) [inline dev-requirements.txt]: note: stubs aren't available for 1.6.0 yet. Wait for them before merging or just make an issue for it?

Comment (classabbyamp) [reply dev-requirements.txt]: available now

**Review by 0x5c — APPROVED** [review]

**Pull request merged**